src/server/system_services/bucket_server.js

```diff
@@ -1683,15 +1675,10 @@ function _calc_metrics({
     };
 
     const actual_free = size_utils.json_to_bigint(_.get(info, 'tiering.data.free') || 0);
-    let available_for_upload = actual_free;
 
-    if (bucket.quota) {
-        let quota_free = size_utils.json_to_bigint(bucket.quota.value).minus(size_utils.json_to_bigint(objects_aggregate.size));
-        available_for_upload = size_utils.size_min([
-            size_utils.bigint_to_json(quota_free),
-            size_utils.bigint_to_json(available_for_upload)
-        ]);
-    }
+    let quota = new Quota(bucket.quota);
+    let available_size_for_upload = Quota.get_available_size_for_upload(quota, actual_free, objects_aggregate.size);
+    let available_quantity_for_upload = Quota.get_available_quantity_for_upload(quota, objects_aggregate.count);
```

Comment:
All those places that we are calling the class directly, Why not use `quota` instead?

Reply:
The methods were static, but I already moved them to the class. Thanks